dojo/forms.py

```diff
@@ -1939,6 +1939,10 @@ class GoogleSheetFieldsForm(forms.Form):
         required=True,
         label="Google Drive folder ID",
         help_text="Extract the Drive folder ID from the URL and provide it here")
+    gmail_address = forms.EmailField(
```

Comment:
I don't think we should specifically call out gmail. I use the feature and use GSuite, as such, I don't use a @gmail.com address.